examples/dbmgr.py

```diff
@@ -181,7 +184,7 @@ def openCloseDatacalc(self):
         """
         if self.openCloseDatacalcResult is not None:
             if not self.openCloseDatacalcResult.done:
-                print("DBMgrApp.openCloseDatacalc(): The previous qiwiscall must be done.")
+                logger.info("DBMgrApp.openCloseDatacalc(): The previous qiwiscall must be done.")
```

Comment:
How about using `logging.warning()`?
If you think `logging.info()` is more appropriate, never mind!

Reply:
I think we can remove the header "DBMgrApp.openCloseDatacalc():" (for the other similar cases as well). Do you agree @BECATRUE?